README.md

```diff
@@ -93,46 +86,31 @@ let g:multi_cursor_skip_key            = '<C-x>'
-
-You can also map your own keys to quit, if ``g:multi_cursor_quit_key`` won't work:
-
-```
-let g:multi_cursor_quit_key='<C-c>'
-nnoremap <C-c> :call multiple_cursors#quit()<CR>
-```
-
 ## Settings
 Currently there are four additional global settings one can tweak:
 
 ### ```g:multi_cursor_exit_from_visual_mode``` (Default: 1)
-If set to 0, then pressing `g:multi_cursor_quit_key` in _Visual_ mode will not quit and delete all existing cursors. This is useful if you want to press Escape and go back to Normal mode, and still be able to operate on all the cursors.
+If set to 0, then pressing `g:multi_cursor_quit_key` in _Visual_ mode will not quit and delete all existing cursors.  
+Useful if you want to go back to Normal mode, and still be able to operate on all the cursors.
 
 ### ```g:multi_cursor_exit_from_insert_mode``` (Default: 1)
-If set to 0, then pressing `g:multi_cursor_quit_key` in _Insert_ mode will not quit and delete all existing cursors. This is useful if you want to press Escape and go back to Normal mode, and still be able to operate on all the cursors.
+If set to 0, then pressing `g:multi_cursor_quit_key` in _Insert_ mode will not quit and delete all existing cursors.  
+Useful if you want to go back to Normal mode, and still be able to operate on all the cursors.
 
 ### ```g:multi_cursor_normal_maps``` (Default: see below)
-Default value: `{'!':1, '@':1, '=':1, 'q':1, 'r':1, 't':1, 'T':1, 'y':1, '[':1, ']':1, '\':1, 'd':1, 'f':1, 'F':1, 'g':1, '"':1, 'z':1, 'c':1, 'm':1, '<':1, '>':1}`
+`{'@': 1, 'F': 1, 'T': 1, '[': 1, '\': 1, ']': 1, '!': 1, '"': 1, 'c': 1, 'd': 1, 'f': 1, 'g': 1, 'm': 1, 'q': 1, 'r': 1, 't': 1, 'y': 1, 'z': 1, '<': 1, '=': 1, '>': 1}`
 
 Any key in this map (values are ignored) will cause multi-cursor _Normal_ mode
 to pause for map completion just like normal vim. Otherwise keys mapped in
-normal mode will "fail to replay" when multiple cursors are active. For example,
-changing it from `{}` to `{'d':1}` makes normal-mode mappings beginning with `d`
-(such as `dw` to delete a word) work in multi-cursor mode.
+normal mode will "fail to replay" when multiple cursors are active.  
+For example: `{'d':1}` makes normal-mode command `dw` work in multi-cursor mode.
 
 ### ```g:multi_cursor_visual_maps``` (Default: see below)
-Default value: `{'i':1, 'a':1, 'f':1, 'F':1, 't':1, 'T':1}`
+`{'T': 1, 'a': 1, 't': 1, 'F': 1, 'f': 1, 'i': 1}`
 
-Any key in this map (values are ignored) will cause multi-cursor _Visual_ mode
-to pause for map completion just like normal vim. Otherwise keys mapped in
-visual mode will "fail to replay" when multiple cursors are active. For example,
-changing it from `{}` to `{'i':1}` makes visual-mode mappings beginning with `i`
-(such as `it` to select an "inner tag block") work in multi-cursor mode.
+Same principle as `g:multi_cursor_normal_maps`
 
 The default list contents should work for anybody, unless they have remapped a
```

Comment:
Given `multi_cursor_visual_maps` now just refers back to `multi_cursor_normal_maps` I think these two paragraphs should probably go under that heading instead.

Reply:
Ok, will do!